deploy/crds/keycloak.org_keycloakrealms_crd.yaml

```diff
@@ -584,6 +584,11 @@ spec:
                     type: string
                 type: object
               type: array
+            unmanaged:
```

Comment:
I think we could get away without using the `unmanaged` field. We could just scan `external.enabled`. If you agree with me, would you mind to remove it?

Reply:
I suggested it earler. External implies it will only ever be used for external things. I think it may be useful to have an unmanaged flag that is a little more general, as you can tell the operator, I'm going to manage this Realm manually, but I still may manage a few clients even when the whole keycloak is otherwise managed by the operator.

Reply:
I see your comment [here](https://github.com/keycloak/keycloak-operator/pull/247#discussion_r488787797). 

The main reason I don't like `unmanaged` and `external.enabled` flags combination is because they result in exactly the same behavior - the Operator doesn't perform reconciliation and obtains Keycloak's URL from `external.url`. But you also have a point - if we add the `external.enabled` flag to the `KeycloakRealm` CR, it will look awkward. Using `managed` flag is much more descriptive.

I'm on the fence with this. But it probably won't hurt to add both flags for the sake of symmetry.